sqlalchemy_redshift/dialect.py

```diff
@@ -228,14 +228,15 @@ class RedshiftDDLCompiler(PGDDLCompiler):
     <BLANKLINE>
     <BLANKLINE>
 
-    Column-level special syntax can also be applied using the column info
-    dictionary. For example, we can specify the ENCODE for a column:
+    Column-level special syntax can also be applied using Redshift dialect
+    specific keyword arguments.
+    For example, we can specify the ENCODE for a column:
 
     >>> product = sa.Table(
     ...     'product',
     ...     metadata,
     ...     sa.Column('id', sa.Integer, primary_key=True),
-    ...     sa.Column('name', sa.String, info={'encode': 'lzo'})
+    ...     sa.Column('name', sa.String, redshift_encode='lzo')
```

Comment:
I'm assuming not all users will immediately be able to use SQLAlchemy 1.3.0+ and they may be confused if they try to run this example with a lower version and encounter an error.

Can we retain a comment here or otherwise indicate that the old `info` syntax is available for older SQLAlchemy?